### ISSUE TITLE
source code errata in chapter 3

### ISSUE THREAD

**Issue by leonardgithub** [PacktPublishing/Building-Machine-Learning-Projects-with-TensorFlow#3]
section: Loss function description and Optimizer loop

now:
fori, j in zip(xvalues, yvalues)

expected:
for i, j in zip(xvalues, yvalues)

**Comment by rnditdev** [comment]
Thank you for detecting the issue, will check it.
